eg-next/examples/textbox-alignment.rs

```diff
@@ -0,0 +1,47 @@
+//! # Example: Multiline text alignment using embedded-text
+//!
+//! This example demonstrates drawing a piece of multiline text that is both horizontally and
+//! vertically centered. The example is using the TextBox from embedded-text. For more examples,
+//! check out the [embedded-text repository](https://github.com/embedded-graphics/embedded-text).
+
+use embedded_graphics::{
+    mono_font::{ascii::FONT_6X10, MonoTextStyleBuilder},
+    pixelcolor::BinaryColor,
+    prelude::*,
+    primitives::Rectangle,
+};
+use embedded_graphics_simulator::{OutputSettingsBuilder, SimulatorDisplay, Window};
+use embedded_text::{
+    alignment::{HorizontalAlignment, VerticalAlignment},
+    style::TextBoxStyleBuilder,
+    TextBox,
+};
+
+fn main() -> Result<(), core::convert::Infallible> {
+    let mut display = SimulatorDisplay::new(Size::new(128, 128));
+
+    let character_style = MonoTextStyleBuilder::new()
+        .font(&FONT_6X10)
+        .text_color(BinaryColor::On)
+        .build();
```

Comment:
No need to use a builder for basic styles:
```suggestion
    let character_style = MonoTextStyle::new(&FONT_6X10, BinaryColor::On);
```